code/modules/telesci/telepad.dm

```diff
@@ -14,8 +14,8 @@
 	..()
 	component_parts = list()
 	component_parts += new /obj/item/circuitboard/telesci_pad(null)
-	component_parts += new /obj/item/ore/bluespace_crystal/artificial(null)
-	component_parts += new /obj/item/ore/bluespace_crystal/artificial(null)
+	component_parts += new /obj/item/stack/ore/bluespace_crystal/artificial(null)
```

Comment:
In another file you add this in the format `component_parts += new /obj/item/stack/ore/bluespace_crystal/artificial(null, 3)` why is it different in this instance?

Reply:
Missed it, fixing

Reply:
Need to fix this still